arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -0,0 +1,572 @@
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/RegistryCredentials"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/regenerateCredentials": {
+      "post": {
+        "tags": [
+          "Registries"
+        ],
+        "description": "Regenerates login credentials for the specified container registry.",
+        "operationId": "Registries_RegenerateCredentials",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The resource group name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "registryName",
+            "in": "path",
+            "description": "The container registry name.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/RegistryCredentials"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RegistryNameCheckRequest": {
+      "description": "The request to check whether the container registry name is available.",
+      "required": [
+        "name",
+        "type"
+      ],
+      "properties": {
+        "name": {
+          "description": "The container registry name.",
+          "type": "string"
+        },
+        "type": {
+          "description": "The container registry resource type.",
+          "enum": [
+            "Microsoft.ContainerRegistry/registries"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ContainerRegistryResourceType",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "RegistryNameStatus": {
+      "description": "The result of checking name availability.",
+      "properties": {
+        "nameAvailable": {
+          "description": "The boolean value that indicates whether the name is available.",
+          "type": "boolean"
+        },
+        "reason": {
+          "description": "The reason that the container registry name could not be used.",
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message that explains the reason value in more detail.",
+          "type": "string"
+        }
+      }
+    },
+    "Registry": {
+      "description": "The container registry.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {

```

Comment:
RegistryProperties is flattened and it will not show up in the generated SDK. How does this matter? For example, the Python SDK generated will have
```
def __init__(self, location, storage_account, tags=None, admin_user_enabled=False):
```
which is correct as location and storage_account are the required properties.